docs/as-a-resource/from-data-to-array.md

```diff
@@ -129,4 +118,5 @@ class AlbumData extends Data
 }
 ```
 
-As always, don't forget to type collections of data objects by annotation or the `DataCollectionOf` attribute, this is essential to transform these collections correctly.
+As always, remember to type collections of data objects by annotation or the `DataCollectionOf` attribute,
```

Comment:
Let's keep these on one line, the spate docs site will make sure they're readable

Reply:
Done 🕺